File: 2019/04/1.jl
#!/usr/bin/julia

f = open("2019/04/test.txt", "r")
inputs = [line for line in readlines(f)]
close(f)

println(inputs)

minmax = [parse(Int, x) for x in split(inputs[1], "-")]

pfirst = []
for x in collect(first(minmax):last(minmax))
    s = string(x)
    c = collect(s)
    if c == sort(c)
        push!(pfirst, s)
    end
end

println(length(pfirst))

psecond = []
for s in pfirst
    for c in s
        if count(isequal(c), s) == 2
            push!(psecond, s)
            break
        end
    end
end

println(length(psecond))

File: 2021/02/1.py
#!/usr/bin/python3

with open('02/input.txt', 'r') as f:
    pos = [0, 0]
    for line in f.readlines():
        cmd, n = line.split()
        n = int(n)
        if cmd == 'forward':
            pos[0] += n
        elif cmd == 'down':
            pos[1] += n
        elif cmd == 'up':
            pos[1] -= n
        else:
            print('BOOBOO')
            break
    print(pos[0] * pos[1])